ui/definitions.d.ts

```diff
@@ -0,0 +1,14 @@
+interface JQueryStatic {
```

Comment:
What does this file do?

Reply:
Provides stub definitions for the JQuery plugins that don't have any available to download.